src/cliffordsynthesis/encoding/SingleGateEncoder.cpp

```diff
@@ -65,7 +101,7 @@ LogicTerm SingleGateEncoder::createSingleQubitGateConstraint(
             (tvars->r[pos + 1] ==
              (tvars->r[pos] ^ tvars->singleQubitRChange(pos, qubit, gate)));
 
-  return changes && createNoChange(pos, qubit, std::nullopt);
```

Comment:
Is the function that this change belongs to even necessary any more after this PR?

Reply:
You are right. That was an oversight.